frontend/src/component/admin/menu/CenteredNavLink.tsx

```diff
@@ -0,0 +1,41 @@
+import { Theme } from '@mui/material';
+import React, { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useTheme } from '@mui/material/styles';
+
+const createNavLinkStyle = (props: {
+    isActive: boolean;
+    theme: Theme;
+}): React.CSSProperties => {
+    const navLinkStyle = {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: '100%',
+        textDecoration: 'none',
+        color: 'inherit',
+        padding: props.theme.spacing(1.5, 3),
+    };
+
+    const activeNavLinkStyle: React.CSSProperties = {
+        fontWeight: 'bold',
+        borderRadius: '3px',
+        padding: props.theme.spacing(1.5, 3),
+    };
+
+    return props.isActive
+        ? { ...navLinkStyle, ...activeNavLinkStyle }
+        : navLinkStyle;
+};
+
+export const CenteredNavLink: FC<{ to: string }> = ({ to, children }) => {
```

Comment:
I wanted to use this one for project tabs but the isActive doesn't work when one of the paths is root path for others (in our case it's Overview that was always selected)